src/smt/smt_engine.cpp

```diff
@@ -263,15 +263,15 @@ void SmtEngine::finishInit()
   // dumping the command twice.
```

Comment:
It seems like this file didn't change at all (or very little?) except for formatting?

Reply:
Indeed, undid all the reformatting for now.